src/libraries/System.IO.FileSystem/tests/File/EncryptDecrypt.cs

```diff
@@ -71,7 +71,14 @@ public void EncryptDecrypt_Read()
                 Assert.Equal(fileContentRead, File.ReadAllText(tmpFileName));
                 Assert.Equal(FileAttributes.Encrypted, (FileAttributes.Encrypted & File.GetAttributes(tmpFileName)));
 
-                File.Decrypt(tmpFileName);
+                // Sometimes Decrypt will fail with, eg.,
+                // System.IO.IOException : The process cannot access the file '...' because it is being used by another process.
+                // Assumption is that it just needs a little more time
+                RetryHelper.Execute(() =>
+                {
+                    File.Decrypt(tmpFileName);
+                }, maxAttempts: 30, backoffFunc: null, retryWhen: e => e.GetType() == typeof(IOException));
```

Comment:
The current largest value on `maxAttempts` is 20, do we really want to try 30 times here?

Reply:
It's a made up number of course, but I upped it a bit because we've really no idea how long it might take. Also because at this point I've touched this test several times to make it more robust !